### PR TITLE
Add standard and runtime hook for pyproj

### PR DESCRIPTION
**Pull request by amifunny**
Fixes pyinstaller/pyinstaller#4828

### CODE REVIEW

File: src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pyproj.py [hunk 1]
```diff
@@ -10,6 +10,32 @@
...
+    tgt_proj_data = os.path.join('Library', 'share', 'proj')
+    src_proj_data = os.path.join(root_path, 'Library', 'share', 'proj')
+
+else:  # both linux and darwin
+    tgt_proj_data = os.path.join('share', 'proj')
+    src_proj_data = os.path.join(root_path, 'share', 'proj')
+
+if os.path.exists(src_proj_data):
+    datas.append((src_proj_data, tgt_proj_data))
+    # a real-time hook takes case to define the path for `PROJ_LIB`
```
Comment (Legorooj) [inline src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pyproj.py:41]:
```suggestion
    # A runtime hook defines the path for `PROJ_LIB`
```

File: src/_pyinstaller_hooks_contrib/hooks/rthooks/pyi_rth_pyproj.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2015-2020, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+#-----------------------------------------------------------------------------
```
Comment (Legorooj) [inline src/_pyinstaller_hooks_contrib/hooks/rthooks/pyi_rth_pyproj.py:10]: This needs to be the Apache header; it's a runtime hook.

File: src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pyproj.py [hunk 3]
```diff
@@ -10,6 +10,32 @@
...
+if is_win:
+    tgt_proj_data = os.path.join('Library', 'share', 'proj')
+    src_proj_data = os.path.join(root_path, 'Library', 'share', 'proj')
+
+else:  # both linux and darwin
+    tgt_proj_data = os.path.join('share', 'proj')
+    src_proj_data = os.path.join(root_path, 'share', 'proj')
+
+if os.path.exists(src_proj_data):
+    datas.append((src_proj_data, tgt_proj_data))
```
Comment (bwoodsend) [inline src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pyproj.py:40]:
```suggestion
from PyInstaller.compat import is_conda
if is_conda:
    if os.path.exists(src_proj_data):
        datas.append((src_proj_data, tgt_proj_data))
    else:
        from PyInstaller.utils.hooks import logger
        logger.warning("Datas for pyproj not found at:\n{}".format(src_proj_data))
```

Comment (bwoodsend) [reply src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pyproj.py:40]: Gives us a heads up if Conda moves anything...

**Comment by Legorooj** [comment]
As I might have said before, in the future _don't_ work on `master`. Work on a feature branch (eg `pyproj-hook`).

**Comment by amifunny** [comment]
@Legorooj  Will keep that in mind

**Pull request merged**